### PR TITLE
chore: add topics

### PR DESCRIPTION
**Pull request by renancaraujo**
## Status

**READY**

## Description

chore: add topics

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: packages/dart_frog_cli/pubspec.yaml [hunk 1]
```diff
@@ -5,6 +5,11 @@ homepage: https://dartfrog.vgv.dev
 repository: https://github.com/VeryGoodOpenSource/dart_frog
 issue_tracker: https://github.com/VeryGoodOpenSource/dart_frog/issues
 documentation: https://dartfrog.vgv.dev/docs/overview
+topics: [server, backend, shelf, dart_frog]
```
Comment (erickzanardo) [inline packages/dart_frog_cli/pubspec.yaml:8]:
```suggestion
topics: [server, backend, shelf, dart_frog, cli]
```

**Review by erickzanardo — DISMISSED** [review]
minor suggestion, but LGTM already

File: packages/dart_frog_gen/pubspec.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ homepage: https://dartfrog.vgv.dev
 repository: https://github.com/VeryGoodOpenSource/dart_frog
 issue_tracker: https://github.com/VeryGoodOpenSource/dart_frog/issues
 documentation: https://dartfrog.vgv.dev/docs/overview
+topics: [server, backend, dart_frog]
```
Comment (wolfenrain) [inline packages/dart_frog_gen/pubspec.yaml:8]: Codegen?

**Review by wolfenrain — DISMISSED** [review]

File: packages/dart_frog_auth/pubspec.yaml [hunk 3]
```diff
@@ -5,6 +5,7 @@ homepage: https://dartfrog.vgv.dev
 repository: https://github.com/VeryGoodOpenSource/dart_frog
 issue_tracker: https://github.com/VeryGoodOpenSource/dart_frog/issues
 documentation: https://dartfrog.vgv.dev/docs/overview
+topics: [server, backend, dart_frog, auth, authentication]
```
Comment (renancaraujo) [inline packages/dart_frog_auth/pubspec.yaml:8]:
```suggestion
topics: [server, backend, dart_frog, auth, authentication, middleware]
```

File: packages/dart_frog_gen/pubspec.yaml (hunk 2 above)
Comment (renancaraujo) [inline packages/dart_frog_gen/pubspec.yaml:8]:
```suggestion
topics: [server, backend, dart_frog, codegen]
```

**Review by scarletteliza — APPROVED** [review]

**Pull request merged**